Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -416,12 +460,14 @@ import { WebClientConfiguration } from "@/models/configData";
 import FeedbackComponent from "@/components/feedback.vue";
 import { library } from "@fortawesome/fontawesome-svg-core";
 import { faStream } from "@fortawesome/free-solid-svg-icons";
+import { BPopover } from "bootstrap-vue";
 library.add(faStream);
 
 const auth: string = "auth";
 const user: string = "user";
 const sidebar: string = "sidebar";
 
+Vue.component("b-popover", BPopover);
```

Comment:
lets add it to the rest of the bootstrap objects in the same location as the rest. I believe its in main.ts or app.vue